webpack/shared.js

```diff
@@ -124,7 +124,7 @@ module.exports = {
 	},
 	plugins: [
 		new ProvidePlugin( {
-			process: 'process/browser',
+			process: 'process/browser.js',
```

Comment:
To be checked. I'm not sure of the impact of this change, though it fixed the errors I was seeing when webpack was compiling through the `npm run dev` command